ovve_ui/utils/comms_link.py

```diff
@@ -302,7 +302,7 @@ def sendPkts(self, cmd_byteData: bytes) -> None:
             self.logger.debug(self.sequenceNoTx)
             self.logger.debug(txCrc)
             self.statPacketTxCntOk+=1
-            self.sequenceNoTx+=1
+            self.sequenceNoTx = (self.sequenceNoTx + 1) % 0xFFFF
```

Comment:
```suggestion
            self.sequenceNoTx = (self.sequenceNoTx + 1) & 0xFFFF
```

I think you mean to bitwise-and this, not mod it (though you could also mod by `0x10000`); since `0xffff % 0xffff = 0`

Reply:
Cool, I learned something!